### ISSUE TITLE
Debugging configuration isn't respected

### ISSUE THREAD

**Issue by smurpau** [kondratyev-nv/vscode-python-test-adapter#185]
So this may not even be an issue, I just can't see where Test Explorer is getting its debugger launch configuration (named "Debug tests") from. Can this be specified so I can edit it, or can it be sourced from the workspace launch.json?

This itself _is_ an issue because when I debug a test (either in Test Explorer or the Python extension's one), logger module handlers printing to stderr only print _once_ for the entire debug session. No idea why, but switching the debugger configuration in .vscode/launch.json: "console" to "externalTerminal" works _for the Python extension test explorer_; but of course not for Test Explorer, because it uses its "Debug tests" configuration from... somewhere.

Any help is greatly appreciated.

Running VS Code v1.48.2, Test Explorer UI 2.19.1, Python Test Explorer v0.4.6, and unittest if that matters.

**Comment by kondratyev-nv** [comment]
@smurpau Thank you for the issue! Indeed, right now `internalConsole` is hardcoded, but other properties are loaded from launch.json file. I will try to fix it so that `console` could be also loaded from launch.json file in the next release.

**Comment by smurpau** [comment]
@kondratyev-nv cheers, the name at least also appears to not be loaded from launch.json. It comes up as "Debug tests" regardless.

**Comment by kondratyev-nv** [comment]
The extension will look for a configuration in launch.json with `"type": "python"` and `"request": "test"` to load any of the following options during debugging
- env
- stopOnEntry
- showReturnValue
- redirectOutput
- debugStdLib
- justMyCode
- subProcess

For example,
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug test configuration",
            "type": "python",
            "request": "test",
            "console": "externalTerminal",
            "justMyCode": false,
            "stopOnEntry": true
        }
    ]
}
```

You can set `name` for the configuration to anything you like. This is just for your convenience, it won't affect UI or any functionality. Also, `console` option is not loaded at the time, but I will fix it in the next couple of days.

**Comment by smurpau** [comment]
The name does appear to affect the UI in the debug console debugger session selector; the name prefixes the module path. I was wrong about it being hardcoded to "Debug tests", it's just "Debug" (the tests comes from my own directory). It isn't of huge significance, it just caused some consternation when it didn't change and 
![image](https://user-images.githubusercontent.com/59456139/92584635-3b991500-f2d7-11ea-8b41-400a1c8a4acd.png)

**Comment by kondratyev-nv** [comment]
Oh, I see. Never noticed that selector. I can change the behavior. So, I can suggest a couple of options.
1. Use the name from the launch.json configuration. In that case, multiple executions will always be in the same session.
2. Use the name from the launch.json configuration as a prefix. So, the session would be named like `<name from configuration> - <test method debugged>`.

I haven't used that functionality in VSCode, so I don't mind any of these options. Do you have any preferences?

**Comment by smurpau** [comment]
Option 2 is the more expected behaviour to me, but I have no strong preference.

**Comment by kondratyev-nv** [comment]
@smurpau I've just published a new version 0.5.0 with loading name and console debug options from launch.json. Please, update and let me know if the issue persists.

**Comment by smurpau** [comment]
@kondratyev-nv I can confirm it all works now, thanks!
